how-to-docs/drone-how-to.md

```diff
@@ -182,7 +182,7 @@ The error points to the missing `DOCKER_PASSWORD` environment variable.
 You can inject the robot's token that has been supplied to you with:
 
 ```
-$ drone secret add --conceal --image="<image_name>" UKHomeOffice/<your_github_repo> DOCKER_PASSWORD your_robot_token
+$ drone secret add --image="<image_name>" --repository ukhomeoffice/<your_github_repo> --name DOCKER_PASSWORD --value your_robot_token
```

Comment:
this is case sensitive. UKHomeOffice

also, you can add secrets via the UI now, might just be worth doing that

Reply:
It wasn't case sensitive when I did it.

Instructions on how to do it through the UI are being added (I think it's in another PR right now). You also have a bit more control if you do it this way since I don't think you can limit secrets for specific events or images through the UI.

Reply:
ah ok cool, it must have changed.